### ISSUE TITLE
Posttest to show up as an intervention

### ISSUE THREAD

**Issue by IvonArroyo** [marshall62/mathspring#263]
David, after all these problems we have had with coordinating the launching of the posttest, we have takled at the affect meeting that we really need to launch the posttest after A CERTAIN AMOUNT OF TIME of use of the system, across days. 
The easiest way to do this, I think, is to launch the posttest as an intervention. popping up a window like the emotion request windows that says "Now we will stop doing math problems for a while. We really want you to answer a series of questions about how much you like math, problem solving and MathSpring in general. Click "continue" below to continue with that activity".
--Ivon

**Comment by IvonArroyo** [comment]
We would still keep the other way of launching a posttest at the beginning of a session, just in case.

**Comment by marshall62** [comment]
No matter what, it's going to have a check in it that says "once you've
seen the post test, you will not be shown a prompt about it again.   So
what went wrong recently was that students were given the post-test but
they didn't do it.   So  a time-based intervention won't solve that
problem.

Also if they are in the tutor and an intervention takes them away to a
post-test page, they may be more likely to blow it off and just go back to
the tutor unless a teacher has specifically told them that "Today we want
you to do a post test, so be on the lookout for it".

Bottom line:  there really is no difference between showing it when they
log in or showing it some time during a tutoring session.    It's probably
more effective at login time because then all students are doing the same
thing at the same time.

I'll build what you are asking for, but I think you're going to find it
more problematic than the one we have.
